### PR TITLE
CKS-693 Update dependencies to fix unit test console errors

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/CKS-693

### CODE REVIEW

File: gatsby/src/components/SiteHeader/SiteHeader.test.tsx [hunk 1]
```diff
@@ -95,77 +100,86 @@ describe("Header", () => {
...
+			const autocompleteSuggestionTemplate = jest.fn(
+				(suggestion) => `<p>${suggestion.Title}</p>`
+			);
+			window.autocompleteSuggestionTemplate = autocompleteSuggestionTemplate;
+
+			renderWithRouter(<SiteHeader />);
+
+			user.type(await waitFor(() => screen.findByRole("combobox")), "dia");
+
+			//TODO need to work out why there are 6 calls being made for the suggestion template versus 3 before GN upgrade
```
Comment (barkertron) [inline gatsby/src/components/SiteHeader/SiteHeader.test.tsx:143]: Have we made a ticket for this?

**Review by barkertron — APPROVED** [review]
All good, just need to make a ticket for that commented out section (sorry if we already have!)

File: gatsby/src/components/SiteHeader/SiteHeader.test.tsx (hunk 1 above)
Comment (johndavey72) [reply gatsby/src/components/SiteHeader/SiteHeader.test.tsx:143]: yes

Comment (johndavey72) [reply gatsby/src/components/SiteHeader/SiteHeader.test.tsx:143]: https://nicedigital.atlassian.net/browse/CKS-701

**Pull request merged**